staticd/static_nb_config.c

```diff
@@ -1296,7 +1296,6 @@ int routing_control_plane_protocols_control_plane_protocol_staticd_segment_routi
 
 	switch (args->event) {
 	case NB_EV_VALIDATE:
-		zlog_info("validating nexthop %pI6", &nexthop.ipaddr_v6);
```

Comment:
Additionally to the bug, this isn't an appropriate log message anyway (if anything it would be debug level and have to be protected by a debug flag).